packages/genesys-spark-components/src/components/beta/gux-icon-tooltip/gux-icon-tooltip.tsx

```diff
@@ -0,0 +1,37 @@
+import { Component, Element, h, JSX, Prop } from '@stencil/core';
+import { trackComponent } from '@utils/tracking/usage';
+import { GuxIconIconName } from '../../stable/gux-icon/gux-icon.types';
+
+/**
+ * @slot content
+ */
+
+@Component({
+  styleUrl: 'gux-icon-tooltip.scss',
+  tag: 'gux-icon-tooltip-beta',
+  shadow: true
+})
+export class GuxIconTooltip {
+  @Element()
+  private root: HTMLElement;
+
+  @Prop()
+  iconName: GuxIconIconName;
+
+  componentWillLoad(): void {
+    trackComponent(this.root);
+  }
+
+  render(): JSX.Element {
+    return (
+      <div class="gux-icon-tooltip">
+        <gux-icon icon-name={this.iconName} size="small" decorative></gux-icon>
```

Comment:
I think you should be able to trigger the tooltip using a keyboard. Maybe on click and also focus. So maybe there should be a button wrapping the icon.

Reply:
@conor-darcy it had been a button in a previous version but @daragh-king-genesys and I felt it was too disruptive for the keyboard user. 

The idea is for the user to focus the field itself and the screenreader to be provided with any additional info at that point. 

However, the tooltip is currently only screenread-able when it's within a focusable element so that's why I created the [accessibility ticket](https://inindca.atlassian.net/browse/COMUI-2895) to make the tooltip text more consistently available to keyboard users.

Reply:
There should be a way to associate that tooltip with the `label` or `input` of the form field. Before we merge this ticket, we need to be sure that this API we have decided on is accessible. I would expect that when I focus a field input, the text that is in the tooltip would be associated with the field input in some way. 

I think it's ok to push out the keyboard accessibility (showing the tooltip on focus) into a separate ticket, but I would advocate for getting the association between the text in the tooltip and the label or input correct in this PR. Feel free to reach out to me and we can try to pair/collaborate on this if you would like.